app/assets/javascripts/pageflow/editor/views/change_theme_view.js

```diff
@@ -0,0 +1,35 @@
+pageflow.ChangeThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'templates/change_theme',
+
+  ui: {
+    changeThemeButton: '.change_theme',
+    labelText: 'label .name'
+  },
+
+  events: {
+    'click .change_theme': function() {
+      pageflow.ChangeThemeDialogView.open({
+        model: this.model,
+        themes: this.options.themes
+      });
+    }
+  },
+
+  onRender: function() {
+    this.ui.labelText.text(this.labelText());
+  },
+
+  labelText: function() {
+    return this.options.label || this.labelPrefix() + ': ' + this.localizedThemeName();
```

Comment:
Scenario for feature 1: I have a customer presentation with a very limited timeframe and want to quickly present the customer with several choices of themes. The customer will be able to see consequences of their choice directly in the editor, together with their media files.

Alternative, without feature 1: The presenter will have to open the theme picker dialog several times or has to resort to a preview that uses static image files, which are in some ways less close to the final product and create needless room for interpretation/questions that way